purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1007,7 +1008,8 @@ class Purchases internal constructor(
             appInBackground,
             { offeringsJSON ->
                 try {
-                    val subscriptionIds = extractSubscriptionIds(offeringsJSON)
+                    val isBC5Enabled = getIsBC5Enabled(offeringsJSON)
```

Comment:
i'm thinking we might want to move the splitting logic at least up to `BillingWrapper`, so it's google-specific. Amazon doesn't need to check for bc5 being enabled (though we _will_ want to be sure that the backend continues to respond as amazon expects...)